### DISCUSSION TITLE
Split 2FA TOTP (pickchars?)

### DISCUSSION THREAD

**Discussion by Sampisa** [keepassxreboot/keepassxc#8135] [Q&A]
I have an authentication type like this one, where TOTP is asked cypher-by-cypher
If the fields have id cypher1 cypher2 cypher3... how can I split the current TOTP in order to put a single cypher on each field?  A "pickchars" can be good too, clicking 1st 2nd 3rd etc. but it should be filled with the current TOTP. And... if it is not possible, is it planned to add it or something similar?

Thank you!

![image](https://user-images.githubusercontent.com/169363/173088432-d817b064-7ee4-4f94-ad4b-e6638930486d.png)

**Comment by droidmonkey** [comment]
Typically dumb sites that do this auto-advance the text cursor between fields (thankfully). In that case you can use a delayed typing of TOTP: `{DELAY=500}{TOTP}`

The browser extension does support these separated fields, but not with an ID starting with cypher. @varjolintu
